srcpkgs/faust/template

```diff
@@ -0,0 +1,54 @@
+# Template file for 'faust'
+pkgname=faust
+version=2.20.2
+revision=1
+build_style=gnu-makefile
+hostmakedepends="cmake pkg-config llvm"
+makedepends="libmicrohttpd-devel llvm-libunwind-devel"
+short_desc="Functional Programming Language for Real Time Signal Processing"
+maintainer="mag <mag-one@autistici.org>"
+license="GPL-2.0-or-later"
+homepage="http://www.faust.grame.fr/"
+distfiles=https://github.com/grame-cncm/faust/releases/download/${version}/${pkgname}-${version}.tar.gz
+checksum="c32588ee60d0f7ca25034d3d5436def6b35495f4d3beb3b02208d48684bcfbe9"
+nostrip_files="libsndfile.so "
+noshlibprovides="libsndfile.so"
+
+case $XBPS_TARGET_MACHINE in
+	*-musl) makedepends+=" libexecinfo-devel";;
+	#armv6l-musl) broken="TODO: fatal error: execinfo.h: No such file or directory";;
+	#aarch64-musl) broken="TODO: fatal error: execinfo.h: No such file or directory";;
```

Comment:
please remove the comments, compilation works for those architectures (i compiled here locally, but didn't test the result)

Reply:
... sorry ... I'm doing mistakes with rebasing ... :( 
Can I close this pull request and reopen a new one?

Reply:
ok, I removed the comments and squashed commits ... It's correct?